src/ResultsPage.js

```diff
@@ -27,11 +27,23 @@ const ResultsPage = ({subjects}) => {
     const includes = queryRoute.get("includes");
     const days = parseDays(queryRoute.get("days"));
     const credits = parseCredits(queryRoute.get("credits"));
-    const totallength = parseLength(queryRoute.get("totallength"))
+    const totallength = parseLength(queryRoute.get("totallength"));
+
     subjects = includes ? subjects.filter(subject => subjectMatches(subject, includes)) : subjects;
-    subjects = days.length > 0 ? subjects.filter(subject => days.includes(subject.day)) : subjects;
-    subjects = subjects.filter(subject => (subject.credits >= credits[0] && subject.credits <= credits[1]));
-    subjects = subjects.filter(subject => (subject.total_len >= totallength[0] && subject.total_len <= totallength[1]));
+
+    // TODO Needs manual fixing
+    // subjects = days.length > 0 ? subjects.filter(subject => days.includes(subject.day)) : subjects;
+    subjects = subjects.filter(subject => 0 <= days.length);  // Avoiding warnings, delete this
```

Comment:
Tohle nechápu ani jako avoid warning.

Reply:
To je výkvět automatické kvality kódu. Ono by to nadávalo na ty `days`, že je nikde nepoužívám, tak jsem je použil (nic to nedělá, pak to smažeme). A asi jsem to mohl všechno zakomentovat, ale ono na to navazuje víc věcí nad tím a pak by mi to možná nadávalo za hodně zakomentovaného kódu :D

Reply:
Jinak to pod // TODO je zakomentovaná ta původní řádka, která teď nefunguje.